### DISCUSSION TITLE
Lesson 4: Problem with node.js

### DISCUSSION THREAD

**Discussion by sl00k** [smartcontractkit/full-blockchain-solidity-course-py#373] [Q&A] [answered]
I can not get the node.js to work

I did download the windows version here https://nodejs.org/en/download/
In Studio Code I get: 
![image](https://user-images.githubusercontent.com/33859801/142019563-71ec2065-ab01-4a68-93b7-f02d8964ec20.png)
Translated:"The naming "node" was not recognized as the name of a cmdlet, function, script file, or executable program. Check the spelling of the name or whether the path is correct (if.)
included) and repeat the process."

While the windows powershell shows:
![image](https://user-images.githubusercontent.com/33859801/142019757-b602fa90-eab4-40e5-9a54-39588ffd2551.png)

**Comment by sl00k** [comment, marked as answer]
omg. coding is so much fun :D 

opening a new terminal in vsc solved it

**Reply by cromewar** [reply]
I'm you solved it so quickly, like all programmer would say, if it does not work, restart it.

**Reply by sl00k** [reply]
the thing i dont get is. restarting vsc or the laptop didnt solve it. seems like vsc saved an old "terminal version"

**Reply by cromewar** [reply]
Yes it does, like a cache version of your previous session so you should  re start the terminal
